client/src/api/upload.js

```diff
@@ -0,0 +1,17 @@
+export const getImageObj = async (name) => {
+  const response = await fetch(`/api/location?name=${name}`);
+  const imgEntry = await response.json();
+  return imgEntry;
+};
+
+export async function postImageObj(image) {
+  const response = await fetch(`/api/upload/`, {
+    method: "POST",
+    body: JSON.stringify(image),
+    headers: {
+      "Content-Type": "application/json",
```

Comment:
not mandatory, but maybe better if you also incoorporate:
```suggestion
      "Content-Type": "application/json;charset=utf-8",
```